vent/api/plugin_helpers.py

```diff
@@ -653,11 +654,13 @@ def start_containers(self,
                     if result[0]:
                         host = result[1]
                     else:
-                        route = Popen(('/sbin/ip', 'route'), stdout=PIPE)
-                        h = check_output(('awk', '/default/ {print $3}'),
-                                         stdin=route.stdout)
-                        route.wait()
-                        host = h.strip()
+                        # get the default device using netifaces
+                        # external library
+                        d_device = netifaces.gateways()
+                        d_device = d_device['default'][netifaces.AF_INET]
+                        host = netifaces.ifaddresses(d_device[1])
+                        host = host[netifaces.AF_INET]
+                        host = host[0]['addr']
```

Comment:
I think there's also an instance of this check in `meta` which still needs to be updated, and this could just called that function from meta.